client/components/checklist/task.js

```diff
@@ -143,9 +145,16 @@ class Task extends PureComponent {
 		return <Gridicon icon={ 'checkmark' } size={ 18 } />;
 	}
 
+	renderChevron = () => {
+		if ( this.isTaskCollapsed() ) {
+			return <Gridicon icon="chevron-down" className="checklist__toggle-affordance" />;
+		}
+
+		return <Gridicon icon="chevron-up" className="checklist__toggle-affordance" />;
```

Comment:
Another way to do this might be to assign the class name based on the conditional check rather than printing out the component twice, e.g., 

```
renderChevron = () =>
    <Gridicon icon={ `chevron-${ this.isTaskCollapsed() ? 'down' : 'up' }` } className="checklist__toggle-affordance" />
```